value.go

```diff
@@ -397,16 +397,19 @@ func (vlog *valueLog) rewrite(f *logFile, tr trace.Trace) error {
 			}
 
 			ne.Value = append([]byte{}, e.Value...)
-			wb = append(wb, ne)
-			size += int64(e.estimateSize(vlog.opt.ValueThreshold))
-			if size >= 64*mi {
+			es := int64(e.estimateSize(vlog.opt.ValueThreshold))
```

Comment:
```suggestion
			es := int64(ne.estimateSize(vlog.opt.ValueThreshold))
```

Reply:
Fixed in https://github.com/dgraph-io/badger/pull/911/commits/294f9d583bf8a5418b89c0eae7a7a0c2a91df345